tests/puppeteer/campaigns/functional/BO/orders/creditSlips/01_createFilterCreditSlips.js

```diff
@@ -25,10 +25,13 @@ const {Statuses} = require('@data/demo/orders');
 let browser;
 let page;
 const today = new Date();
+const day = (`0${today.getDate()}`).slice(-2); // Current day
+const month = (`0${today.getMonth() + 1}`).slice(-2); // Current month
```

Comment:
Hello !

Just a question: it seems that you add a zero all the time, what happens if the current day is, let's say 15 ? Will it work?

Reply:
Hello!
Yes, it works because slice (-2) extracts the two last elements in the sequence.